test-cases/performance/perf-regression.100threads.30M-keys.yaml

```diff
@@ -21,3 +21,5 @@ backtrace_decoding: false
 store_perf_results: true
 send_email: true
 email_recipients: ['qa@scylladb.com']
+
+use_legacy_cluster_init: false
```

Comment:
@Roy, this PR use latest master, so all code is already in. This config enable the valid cluster init.

Reply:
Ok, but I need the PR on top of branch-perf-v8.

Didn’t it work straightforward on master? The non legacy is default there.

Also, I saw that @dkropachev used 4.3.rc1 with branch-perf-v9 on his LWT perf job.
Check with him if it worked without any fix to run it.